core-bundle/contao/controllers/BackendPopup.php

```diff
@@ -209,4 +218,90 @@ public function run()
 
 		return $objTemplate->getResponse();
 	}
+
+	private function buildMetadataTables($path): array
+	{
+		$tables = array();
+
+		$metadata = System::getContainer()->get('contao.image.metadata')->parse($path)->getAll();
+
+		foreach ($metadata as $format => $data)
+		{
+			$table = array();
+
+			foreach ($data as $category => $rows)
+			{
+				if (!\is_array($rows) || array_is_list($rows))
+				{
+					if ($format === 'iptc' && (str_starts_with($category, '1#') || str_ends_with($category, '#000')))
+					{
+						continue;
+					}
+					$table[] = array(
+						'path' => StringUtil::specialchars($format . ':' . $category),
+						'label' => StringUtil::specialchars($format === 'iptc' ? $this->iptcLabel($category) : ucfirst($category)),
+						'value' => StringUtil::specialchars($this->arrayToString((array) $rows)),
+					);
+				}
+				else
+				{
+					foreach ($rows as $label => $value)
+					{
+						if ($format === 'xmp' && $label === 'about' && !$value)
+						{
+							continue;
+						}
+						$table[] = array(
+							'path' => StringUtil::specialchars($format . ':' . $category . ':' . $label),
+							'label' => StringUtil::specialchars(ucfirst($label)),
+							'value' => StringUtil::specialchars($this->arrayToString((array) $value)),
+						);
+					}
+				}
+			}
+
+			$tables[StringUtil::specialchars(strtoupper($format === 'iptc' ? 'IPTC IIM' : $format)) . ' ' . $GLOBALS['TL_LANG']['MSC']['fileMeta']] = $table;
+		}
+
+		return array_filter($tables);
+	}
+
+	private function arrayToString(array $values): string
+	{
+		$values = array_map(
+			fn ($value) => \is_array($value) ? $this->arrayToString($value) : $value,
+			$values,
+		);
+
+		return implode(', ', $values);
+	}
+
+	private function iptcLabel(string $key): string
```

Comment:
Looks like something that could go to `contao/image`? 🤔

Reply:
I’m not sure. Providing proper labels does not have to be part of the library I think. If it would be it should IMO work for all metadata formats (and be more complete) and that is properly way too much work.

I just see this as an UX improvment for your backend users.

Reply:
Yeah which I think is definitely part of the library. It doesn't have to be translated or anything as it is mainly technical. But `City` is always better than `2#090`, right? If I'm using `contao/image` in another project, I would want `City` there too 😇

Reply:
Added a `toReadable()` method in https://github.com/contao/image/pull/96